.travis.yml

```diff
@@ -48,6 +48,8 @@ matrix:
       env: TOXENV=txchecker-travis-required
     - python: 2.7
       env: TOXENV=txchecker-travis-all
+    - python: 2.7
```

Comment:
I think we should not make Treq able to fail the build; it relies on external websites (httpbin?) for its integration testing. Of course, that hasn't been an issue on treq development, so... use your discretion.